CONTRIBUTING.md

```diff
@@ -156,7 +156,6 @@ You should have [ESLint running in your editor](http://eslint.org/docs/user-guid
 
 > Please do not ignore any linting errors, as they are meant to **help** you and to ensure a clean and simple code base.
 
-Make sure none of your JavaScript is longer than 80 characters per line. The reason we enforce this is because one of our dependent NPM modules, [jsonlint](https://github.com/zaach/jsonlint), does not fully support wildcard paths in Windows.
```

Comment:
I think it is more historical, the 80char rule tripped up many contributor because we weren't running `npm test` before pushing a commit, which annoying broke the travis build.

We couldn't default everyone to  `npm test` because of the jsonlint/windows issue, so we mentioned the 80char rule here instead 

I guess if we are all now doing `npm test`, it can be removed :+1: